modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -42,38 +63,16 @@ def merge_clinics(appointments)
           end
         end
 
-        def merge_facilities(appointments)
-          cached_facilities = {}
-          appointments.each do |appt|
-            facility_id = appt[:location_id]
-            next unless facility_id
-
-            cached = cached_facilities[facility_id]
-            cached_facilities[facility_id] = get_facility(facility_id) unless cached
-            appt[:location] = cached_facilities[facility_id]
-          end
-        end
-
         def get_clinic(location_id, clinic_id)
-          clinics = v2_systems_service.get_facility_clinics(location_id: location_id, clinic_ids: clinic_id)
-          clinics.first unless clinics.empty?
-        rescue Common::Exceptions::BackendServiceException
+          vaos_mobile_facility_service.get_clinic(station_id: location_id, clinic_id: clinic_id)
```

Comment:
did not need to even extract the vista id out of the previous call since the `get_clinic` method takes the first 3 characters automatically.